doc/make.bat

```diff
@@ -29,7 +29,7 @@ if errorlevel 9009 (
 	exit /b 1
 )
 
-python settings_rstgen.py
+REM python settings_rstgen.py
```

Comment:
Remove?

Reply:
The GitHub is failing while reading `settings` rst files hence we have commented this for now.

We are going to uncomment this later on.

Reply:
Yeah, same as above.  I'm not sure the rst generation should be done here right.  I think for the TUI it's happening elsewhere, as part of the calls in `codegen`.

Reply:
@dnwillia-work 

We have verified that we are able to generate `datamodel`, `TUI` and `settings` html docs locally without any error.
We have updated the code and reverted above change. Thank you.

![image](https://user-images.githubusercontent.com/106588300/218015480-a8fb32de-32a5-4fa2-9c4c-f1a1723e9bad.png)

`---------------------------------------------------------------------------------------------------------------------`

![image](https://user-images.githubusercontent.com/106588300/218015577-ba901fc4-f5ae-4b44-a123-5241dd5c76cf.png)

`---------------------------------------------------------------------------------------------------------------------`

![image](https://user-images.githubusercontent.com/106588300/218015718-a009ceb1-68e7-4182-8b77-7e4bb5d9e30b.png)